rules/readme-has-name.js

```diff
@@ -60,11 +60,11 @@ function readmeHasName(tree, file) {
 
 	h1Content = h1Content.trim()
 
-	if (h1Content == bowerJson.name) {
+	if (h1Content == packageJson.name) {
 		return
 	} else {
 		file.message(
-			`The main heading in the README must be the component's name as defined in the bower.json. expected "${bowerJson.name}", got "${h1Content}"`,
+			`The main heading in the README must be the component's name as defined in the package.json. expected "${packageJson.name}", got "${h1Content}"`,
```

Comment:
@Financial-Times/origami-core, should we be checking for the full package name which includes the namespace, or only the part after the namespace?

The code currently checks for the full package name. E.G. It wants `@financial-times/o-errors` and not `o-errors` as the main heading.

![Screenshot of a warning being emitted because the heading is `o-errors` and not `@financial-times/o-errors`](https://user-images.githubusercontent.com/1569131/113856510-4185a480-9799-11eb-8f06-b4c4876ff5c0.png)

Reply:
We agreed in the origami office hours to not include the `@finanicial-times/`